lib/iris/tests/test_image_json.py

```diff
@@ -29,55 +29,45 @@
 import json
 import os
 import requests
-import unittest
-import time
 
 
 @tests.skip_inet
-@tests.skip_data
 class TestImageFile(tests.IrisTest):
     def test_resolve(self):
-        # https://developer.github.com/v3/#user-agent-required
-        headers = {'User-Agent': 'scitools-bot'}
-        rate_limit_uri = 'https://api.github.com/rate_limit'
-        rl = requests.get(rate_limit_uri, headers=headers)
-        some_left = False
-        if rl.status_code == 200:
-            rates = rl.json()
-            remaining = rates.get('rate', {})
-            ghapi_remaining = remaining.get('remaining')
-        else:
-            ghapi_remaining = 0
+        listingfile_uri = (
+            'https://raw.githubusercontent.com/SciTools/test-iris-imagehash'
+            '/gh-pages/v4_files_listing.txt')
+        req = requests.get(listingfile_uri)
+        if req.status_code != 200:
+            raise ValueError('Get failed on image listings file: {}'.format(
```

Comment:
You could capitalise `GET` to make it clear it is a http request. Otherwise, the error reads a bit funnily out of context

Reply:
Good idea, I see what you mean :grin: